src/indicators/idmc.R

```diff
@@ -0,0 +1,98 @@
+library(idmc)
+library(tidyverse)
+
+output_dir <- file.path(
+  Sys.getenv("AA_DATA_DIR"),
+  "private",
+  "exploration",
+  "glb",
+  "global_monitoring",
+  "outputs"
+)
+
+##############
+#### IDMC ####
+##############
+
+df_idmc_raw <- idmc_get_data()
+
+df_idmc <- df_idmc_raw %>%
+  idmc_transform_daily() %>%
+  idmc_rolling_sum() %>%
+  idmc_flagging()
+
+# flagging
+
+df_idmc_flags <- df_idmc %>%
+  select(
+    -flag_total
+  ) %>%
+  rename_with(
+    .fn = ~gsub("(flag_)(.*)", "\\1idmc_\\2", .x),
+    .cols = starts_with("flag")
+  ) %>%
+  pivot_longer(
+    cols = starts_with("flag"),
+    names_to = "flag_type",
+    values_to = "flag"
+  ) %>%
+  select(
+    -starts_with('displacement')
+  ) %>%
+  group_by(
+    iso3,
+    country,
+    flag_type
+  ) %>%
+  filter(
+    flag
+  ) %>%
+  mutate(
+    cs_temp_ = cumsum(date - lag(date, default = min(date)) != lubridate::days(1))
```

Comment:
I get an error here: 
> Problem while computing `cs_temp_ = cumsum(date - lag(date, default = min(date)) != lubridate::days(1))`.
ℹ The error occurred in group 1: iso3 = "AB9", country = "Abyei Area", flag_type = "flag_idmc_1st_3_months".
Caused by error in `date - lag(date, default = min(date))`: